### PR TITLE
fix input error

### PR DESCRIPTION
**Pull request by aYishinHo**
### Summary
- fix `Input` error
- handle value formatter `onChange`

### CODE REVIEW

File: ui/src/components/Input/Input.tsx [hunk 1]
```diff
@@ -26,7 +26,7 @@ export default function Input({
     <div className={style.inputContainer}>
       {label && <label>{label}</label>}
       <div className={style.input}>
-        <input name={name} type={type} value={value} onChange={(e) => onChange(e)} placeholder={placeholder} />
+        <input name={name} type={type} value={value || ''} onChange={(e) => onChange(e)} placeholder={placeholder} />
```
Comment (aStanleyLiang) [inline ui/src/components/Input/Input.tsx]: It's not necessary to dynamically assign a function in props onChange. OnChange can be assgined to the props directly:
```
onChange={onChange}
```

**Review by aStanleyLiang — CHANGES_REQUESTED** [review]
overall looks good, minor changes is required.

**Review by aStanleyLiang — APPROVED** [review]

File: ui/src/components/Input/Input.tsx (hunk 1 above)
Comment (aYishinHo) [reply ui/src/components/Input/Input.tsx]: fix in 569812e

**Review by aMichaelPoernomo — APPROVED** [review]
subarashi~

**Pull request merged**